### ISSUE TITLE
Definition Requests

### ISSUE THREAD

**Issue by MysteryGodzilla** [Yata-Dash/Yata-Dash#5]
These trackers I have had requests to add so I will work on next

## CURRENTLY SUPPORTED 

| Tracker | Platform | Approved | Stats | Limit | Notes |
|---|---|---|---|---|---|
| Aither | UNIT3D | Yes | 7/7 | API only |  |
| AnimeBytes | Custom API | No | 4/4 | API only | Uses the personal stats API; account age is entered manually |
| Anthelion | Gazelle (ANT/NEB) | Yes | 6/6 | API only | Added API stats - Thanks team! |
| Blutopia | UNIT3D | No | 3/5 | API only |  |
| BroadcastTheNet | Custom API | No | 5/6 | API only | JSON-RPC userInfo; 150 API calls per hour |
| DarkPeers | UNIT3D | Yes | 3/6 (4 scraped) | 180min |  |
| GazelleGames | GazelleGames | No | 1/1 | API only |  |
| Hawke-uno | UNIT3D | No | 6/6 | API only | Not on this tracker can't seek approval |
| HHD | UNIT3D | No | 6/6 | API only |  |
| InfinityHD | UNIT3D | Yes | 3/6 (4 scraped) | Default |  |
| LST | UNIT3D | Yes | 3/6 (4 scraped) | 180min |  |
| Luminarr | UNIT3D | Yes | 3/6 (4 scraped) | 120min |  |
| MidnightScene | UNIT3D | Yes | 3/5 (4 scraped) | 60min |  |
| MyAnonamouse | Custom API | Yes | 3/3 | API only |  |
| Nebulance | Gazelle (ANT/NEB) | No | 5/5 | API only | Ratioless; episode and season seed-time rules differ |
| OldToonsWorld | UNIT3D | Yes | 5/5 | API only | Added all required stats to API - Thanks team! |
| OnlyEncodes+ | UNIT3D | Yes | 3/6 (4 scraped) | Once per day |  |
| Orpheus | Gazelle (ajax.php) | No | 6/6 | API only | Required ratio is calculated dynamically |
| PeerGarden | traxary | No | 4/6 | API only |  |
| Redacted | Gazelle (ajax.php) | No | 6/6 | API only |  |
| ReelFliX | UNIT3D | Yes | 6/6 | API only |  |
| RetroFlix | Custom API | Yes | 3/3 | API only | Added API stats - Thanks team! |
| RocketHD | UNIT3D | Yes | 3/6 | API only |  |
| Seedpool | UNIT3D | Yes | 2/3 (3 scraped) | 180min |  |
| SpeedApp | Custom API | Yes | 3/3 | API only | Added API stats - Thanks team! |
| Unwalled | UNIT3D | Yes | 3/5 (4 scraped) | 180min |  |
| Upload.cx | UNIT3D | No | 3/6 | API only |  |
| YUSCENE | UNIT3D | Yes | 3/5 (4 scraped) | 180min |  |
| Zenith | UNIT3D | Yes | 6/6 | API only | API reworked: extended stats and events, proposed upstream |

#### These were added on info alone or by a third party. I can not gurantee these are correct or approved. If you are staff from one of these trackers please reach out if you want to approve/remove or work with me on this.
Huno - added from external info - no approval dont yet have access to seek approval
AnimeBytes - added by gizzlepox
Blutopia - added by gizzlepox
BroadcasTheNet - added by gizzlepox
GazelleGames - added by gizzlepox
Nebulance - added by gizzlepox
Orpheus - added by gizzlepox
Redacted - added by gizzlepox
Upload.cx - added by gizzlepox

### IN PROGRESS
Traxary based trackers

#### If staff want to provide access for the below or create the def themself, I'm happy to push your tracker through
BHD - requested by user, no staff approval or access to tracker yet
DVL [UNIT3D] - requested by user, no staff approval or access to tracker yet
FZ [UNIT3D] - requested by user, no staff approval or access to tracker yet
GF [UNIT3D] - requested by user, no staff approval or access to tracker yet
NQ [UNIT3D] - requested by user, no staff approval or access to tracker yet
RMC [UNIT3D] - requested by user, no staff approval or access to tracker yet
STC [UNIT3D] - requested by user, no staff approval or access to tracker yet
TH [UNIT3D] - requested by user, no staff approval or access to tracker yet
HD-Space [XBTIT] - requested by user, no staff approval or access to tracker yet
CrabPT [NexusPHP] - requested by user, no staff approval or access to tracker yet

### SESSION TOKEN APIs - WANT APPROVAL OR OPEN ENDPOINT BEFORE ADDING
Digitalcore - API access for user stats behind cookie auth, would prefer bearer endpoint
AlphaRatio - requested by user, no staff approval yet
GreatPosterWall - requested by user, no staff approval yet

### NO API OR UNLIKELY TO BE ADDED ANYTIME SOON - NO REQUESTS UNLESS FROM STAFF
Broken stones [GAZELLE] - RSS only no API, would require scrape only
All Z trackers
TL
IPT

**Comment by d0ofy** [comment]
Please add MNS to the list

**Comment by gizzlepox** [comment]
AR, BTN, GGn, BLU, RFX and ULCX also have APIs and could be added. :)

**Comment by MysteryGodzilla** [comment]
> AR, BTN, GGn, BLU, RFX and ULCX also have APIs and could be added. :)

Added to the request list - some of these I have open invite paths for so I will prioritise those first. Others I don't have access to so will need external help or staff to assist until I can gain access.

**Comment by ThatNeoByte** [comment]
DVL, FZ, GF, NQ, RMC, STC, TH all expose the default Unit3D api, and of which scraping works correctly.

**Comment by MysteryGodzilla** [comment]
> DVL, FZ, GF, NQ, RMC, STC, TH all expose the default Unit3D api, and of which scraping works correctly.

Scraping is default off since some staff and trackers don't like and actively ban from it. Individuals use at your own risk. When I eventually get access to trackers I ask staff for approval to scrape or ideally get work on the API to remove the need for scraping.
Defs are also preferable so user group info and tracker rules (eg min ratio) are passed through.

I do pass all scraping through a UA identifying as Yata-dash so trackers can see and track traffic specific to this app. This helps trackers manage how much traffic its costing them, but also help them identify if they haven't allowed it.

Added those trackers to the pending list
